Feature/task7

HubertKarw opened the pull request.

HubertKarw committed:
java/product-project/task6/formatting

HubertKarw committed:
java/product-project/task7/created ComandLine Interface
created interface for comandLineImplementation

HubertKarw committed:
java/product-project/task7/formatting

HubertKarw committed:
java/product-project/task7/commandLineImpl class
Added commandline implementation class and some of its functionality

HubertKarw committed:
java/product-project/task7/command line interface changes

HubertKarw committed:
Merge branch 'master' into feature/task7

# Conflicts:
#	src/com/company/Cart.java

HubertKarw committed:
java/product-project/task7/remove CommandLineInterface.java

HubertKarw committed:
java/product-project/task7/addToCart and removeFromCart implementation in cmdLineImpl

HubertKarw committed:
java/product-project/task7/changes in functionality
added order processing, order creation and changes some functionality to count products with their stock in order

HubertKarw committed:
java/product-project/task7/formatting and new methods in interface

kkpolak commented on src/com/company/CommandLineImpl.java:
```diff
@@ -0,0 +1,46 @@
+package com.company;
+
+public class CommandLineImpl implements CommandLine {
+    public CommandLineImpl() {
```
czy potrzebny nam taki konstruktor, nie wolimy skorzystac z domyslnego?

kkpolak commented on src/com/company/CommandLineImpl.java:
```diff
@@ -0,0 +1,46 @@
+public class CommandLineImpl implements CommandLine {
+    public CommandLineImpl() {
+    }
+
```
Tutaj musimy miec wiecej interakcji z uzytkownikiem ktory odpala program, typu:
-> uzytkownik widzi przyciski: 
- pokaz liste produktow, 
- pokaz koszyk, 
- dodaj produkt do koszyka
- zloz zamowienie
- ...

-> wybiera liste produktow -> pokazuje sie lista produktow
-> to samo koszyk jak wyzej^

-> wybiera dodaj produkt -> dostaje napis: 'podaj id produktu', -> podaje id -> produkt dodaje sie do koszyka

to podstawowe scenariusze, mozna oczywiscie rozszerzyc

HubertKarw committed:
java/product-project/task7/!commendline with scanner implementation! - not working

HubertKarw committed:
java/product-project/task7/commendline with scanner implementation
fixed not working scanner problem
added scanner implementatrion of adding and removing from cart methods

HubertKarw committed:
java/product-project/task7/formatting

HubertKarw committed:
java/product-project/task7/remove redundant UUID CAST
